src/environments/environment.prod.ts

```diff
@@ -0,0 +1,3 @@
+export const environment = {
```

Comment:
What prompted the need for these environment files?

Reply:
This answer will be for this file, for `src/main.ts`, `src/polyfill.ts` and both `tsconfig` files. 
They are needed for the angular CLI to work. The main purpose of these changes is to make the `ng` commands of its work.
You can use `npm run ng` to access them. I didn't dive in deep why any specific of these files are necessary, but removing them caused some problems when running the `ng` commands.